custom-ci-cd-environment-with-docker_5caf3f432c7d3a392f9ceb50.md

```diff
@@ -108,11 +108,16 @@ on the `db` hostname in the first container.
 ## Pre-built convenience Docker images for Semaphore CI/CD jobs
 
 For convenience, Semaphore comes with a [repository of pre-built images hosted
-on Dockerhub][dockerhub-semaphore]. The source code of the Semaphore Docker
-images is [hosted on Github][docker-images-repo].
+on Dockerhub](https://hub.docker.com/u/semaphoreci). The source code of the Semaphore Docker
+images is [hosted on Github](https://github.com/semaphoreci/docker-images).
```

Comment:
These were reference style links, as requested and documented in the CONTRIBUTING document https://github.com/semaphoreci/docs/blob/master/CONTRIBUTING.md#text-formatting.

Reference style links: https://www.markdownguide.org/basic-syntax/#reference-style-links